macros/models/_sector/nft/platforms/seaport_v3_trades.sql

```diff
@@ -285,7 +285,7 @@ with source_ethereum_transactions as (
                   ,contract_address as platform_contract_address
             from (select *
                     from {{ Seaport_call_matchOrders }}
-                    cross join unnest(output_executions) with ordinality as foo(execution,execution_idx)
+                    cross join unnest(output_0) with ordinality as foo(execution,execution_idx)
```

Comment:
@jeff-dude I don't think we should put `output_0` (or `_0`, `_1`) arguments in the macros, we should have the proper ABIs that give us the correct names for input and output arguments. 
So I believe this should be fixed in the decode step, not in spellboook.

Reply:
my understanding after talking with decode team is that the fix isn't as straightforward as we'd like. this PR is aiming for short-term fixes across the board, so i agree that ideally we follow up on this quickly and fix for the long term. this unblocks us for the time being

Reply:
we will follow up asap, @aalan3 and i think we should have all these things prioritized for next week, with you leading the effort / us supporting where needed

Reply:
this won't work in macros anyway without if/else logic 🫠

Reply:
@0xRobin this is a temporary fix so that we can get through the weekend without removing these models. We will have to fix this next week. 

Regarding the names, it appears that the latest ABI for [this contract](https://etherscan.io/address/0x0000000000000068f116a894984e2db1123eb395#code) has removed the names? Otherwise if this contract is not relevant we can look into removing it and reverting.